### ISSUE TITLE
exp_reweight = exp_reweight / np.sum(exp_reweight) * num_foreground

### ISSUE THREAD

**Issue by Liu-wanbing** [Megvii-BaseDetection/DisAlign#6]
Dear author, I have some questions about the code and paper:
1. `exp_reweight = exp_reweight / np.sum(exp_reweight) * num_foreground`
    Why "exp_reweight" is  multiplied by the coefficient "**num_foreground**"? It is not mentioned in the paper. 
2.  Is "**K**" in _the empirical class frequencies r = [r1, · · · , rK] on the training set_ in the paper the same as the class number C of the training set?

**Comment by tonysy** [comment]
Hi, 
1. num_foreground is used to maintain the total loss coefficient same with the no-reweight setting. It's a common operation in reweight-based methods.
2. exp_reweight is the class frequencies in the training set. I will upload the class frequency files, or you can calculate based on the annotation of the training set.

Thanks.

**Comment by Liu-wanbing** [comment]
Thanks for your reply!!
